redisdb/datadog_checks/redisdb/redisdb.py

```diff
@@ -241,6 +243,16 @@ def _check_db(self):
             elif info_name in self.RATE_KEYS:
                 self.rate(self.RATE_KEYS[info_name], info[info_name], tags=tags)
 
+        clients_by_name = defaultdict(int)
+        clients = conn.client_list()
+        for c in clients:
+            name = c["name"]
+            if name == "":
+                name = DEFAULT_CLIENT_NAME
+            clients_by_name[name] += 1
+        for name, count in clients_by_name.items():
+            self.gauge("redis.net.connections", count, tags=tags + ['source:' + name])
```

Comment:
Should we move this after the `for config_key, value...` block below? That block seems related to "db-wide statistics" too, so it's probably best to keep it close to the previous block.

Then also maybe add a small comment to indicate what this block is doing? Something like…

```python
# Save client connections statistics
```

Reply:
Ah yes you're right